packages/sshnoports/scripts/install_sshnpd

```diff
@@ -38,7 +38,7 @@ usage() {
   echo "      --args <args>           Additional arguments to sshnpd ("-v" by default)"
   echo "      Possible args:"
   echo "        -s, --[no-]sshpublickey      Update authorized_keys to include public key from sshnp"
-  echo "        -u, --[no-]username          Send username to the manager to allow sshnp to display username in command line"
+  echo "        -u, --[no-]un-hide          When set, makes various information visible to the manager atSign - e.g. username, version, etc"
```

Comment:
(Sorry!) Missing a space ... help text output would not be aligned.

Reply:
Good catch, thanks